site/admission/index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <title>Your Website Name</title>
+  <link rel="stylesheet" href="/assets/css/reset.css" type="text/css" />
+  <link rel="stylesheet" href="/assets/css/global.css" type="text/css" />
+  <link rel="stylesheet" href="styles.css" type="text/css" />
+</head>
+
+<body>
+  <nav><a href="/">Home</a><a href="/about">About</a></nav>
+  <h1>Admission</h1>
+  <p>Jodie McSally Dance Program, Class of 21</p>
+  <div>
+    <div>Overview</div>
+    <div>Application Process</div>
+    <div>Auditions</div>
+    <div>FAQ</div>
+  </div>
+  <div>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+      magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut.</p>
+  </div>
+  <h5>Students must live in Boston by TKTK in order to enroll. Learn more</h5>
+  <div>
+    <div>Grades Blind</div>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing
+      elit, sed do eiusmod tempor incididunt ut labore et
+      dolore magna aliqua. Ut enim ad minim.</p>
+  </div>
+  <div>
+    <div>All Levels of Experience</div>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing
+      elit, sed do eiusmod tempor incididunt ut labore et
+      dolore magna aliqua. Ut enim ad minim.</p>
+  </div>
+  <div>
+    <div>Full Inclusion</div>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing
+      elit, sed do eiusmod tempor incididunt ut labore et
+      dolore magna aliqua. Ut enim ad minim.</p>
+  </div>
+  <h3>Upcoming Admissions Events</h3>
```

Comment:
Try using the same header level for headers that are similar. For example, when I look at the UX PDF both "Upcoming Admissions Events" and "Application Process" look like the same hierarchy to me. What this means is they both introduce a section on the page. In that case I would use the same h tag for those. You could make them both `<h2>` for example.